salt/client/ssh/state.py

```diff
@@ -41,13 +41,15 @@ def load_modules(self, data=None, proxy=None):
         Load up the modules for remote compilation via ssh
         """
         self.functions = salt.loader.ssh_wrapper(self.opts, None, context=self.context)
+        if self.wrapper is not None:
+            self.wrapper.wfuncs = self.functions
         self.utils = salt.loader.utils(self.opts)
         self.serializers = salt.loader.serializers(self.opts)
         locals_ = salt.loader.minion_mods(self.opts, utils=self.utils)
         self.states = salt.loader.states(
             self.opts, locals_, self.utils, self.serializers
         )
-        self.rend = salt.loader.render(self.opts, self.functions)
+        self.rend = salt.loader.render(self.opts, self.wrapper)
```

Comment:
This fixes my latest comment in this PR.

Reply:
Yes, however this reverts the fix in https://github.com/saltstack/salt/pull/61634.  This is likely what we will end up doing for now and revisiting the aforementioned issue later, but just so you are aware.